src/App.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import HomePage from './components/HomePage';
-import CategoriesPage from './components/CategoriesPage';
+import Categories from './Components/Categories';
+import Home from './Components/Home';
+import Navbar from './Components/Navbar';
```

Comment:
- [ ] Kindly make sure that the App running successfully When I run the App I got the below error related with spelling error when you import the modules from the **components folder** when You rename your folder  you make the first **letter capital C for the components kindly make it small letter c just like the components**.
![spelling](https://github.com/sam2if/Bookstore/assets/93598189/72ecd89b-9ce8-422d-bd37-7d4af9c3ce6c)